server/.env

```diff
@@ -1,2 +1,3 @@
 TEAM_NAME="Shums, Kareem, Gabriel, Moshe"
 FLASK_ENV="development"
+GOOGLE_API_KEY = "AIzaSyAN3q6E8D80h38QQsiSWQ5keW6YSRMPqYc"
```

Comment:
This is a public facing key right? If it's the private one, please remove this, and we will need to refresh this key